commit 168d82ddd0c3cb749978f8057a85cc077b8df8d2
Author: Khem Raj <raj.khem@gmail.com>
Date:   Fri Feb 10 17:57:08 2017 -0800

    build/arm64: Add rules for .dtbo files for dts overlays
    
    We now create overlays as .dtbo files.
    
    Signed-off-by: Khem Raj <raj.khem@gmail.com>

diff --git a/arch/arm64/Makefile b/arch/arm64/Makefile
index 939b310913cf..8d90800c10f9 100644
--- a/arch/arm64/Makefile
+++ b/arch/arm64/Makefile
@@ -131,6 +131,9 @@ zinstall install:
 %.dtb: scripts
 	$(Q)$(MAKE) $(build)=$(boot)/dts $(boot)/dts/$@
 
+%.dtbo: | scripts
+	$(Q)$(MAKE) $(build)=$(boot)/dts MACHINE=$(MACHINE) $(boot)/dts/$@
+
 PHONY += dtbs dtbs_install
 
 dtbs: prepare scripts
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1306_168d82ddd0c3cb749978f8057a85cc077b8df8d2.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1306_168d82ddd0c3cb749978f8057a85cc077b8df8d2.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
